source/date_time.md

```diff
@@ -0,0 +1,112 @@
+
+## How to set up Date and Time question types
+
+### Setting up in Formbuilder
+
+Adding “Date”, “Time”, and “Date and Time” questions is simple:
+
+-   In the Formbuilder, click the <i class="k-icon k-icon-plus"></i> button to
+    add a new question
+-   Type the question text, for example “What is your date of birth?”, then
+    click **ADD QUESTION** or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding the questions](images/date_time/adding.gif)
+
+### Setting up in XLSForm
+
+To add "Date", "Time", and "Date and Time" questions in the XLSForm, use the
+`date`, `time` and `dateTime` question types as shown in the example below:
+
+In XLSForm, you can set up the following:
+
+| type     | name      | label                                         |
+| :------- | :-------- | :-------------------------------------------- |
+| date     | dob       | On what date were you born?                   |
+| time     | time      | At what time do you you leave for work?       |
+| dateTime | date_time | At what date and time did the training start? |
+
+## Appearance of date and time question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances](images/date_time/default_appearances.png)
+
+### Advanced appearances
+
+When adding the “Date” question type in the Formbuilder, you can choose from a
+number of display options (under the question settings). Appearances change the
+way the question is displayed on web forms and on KoboCollect.
+
+![Adding advanced appearances](images/date_time/advanced_appearance.png)
+
+To add appearance values that are not listed on the drop-downlist in the
+Formbuilder, choose "other", and type the appearance value in the text field
+that appears.
+
+![Advanced appearances](images/date_time/advanced_appearances.png)
+
+### Adding custom appearances for date questions in XLSForm
+
+You can specify advanced appearances in XLSForm through the appearance column as
+follows:
+
+| type | name                | label                                   | appearance     |
+| :--- | :------------------ | :-------------------------------------- | :------------- |
+| date | rains_start         | When did the planting rains start?      | month-year     |
+| date | year_migrate        | In which year did you migrate?          | year           |
+| date | no-calendar_date    | Date picker without calendar            | no-calendar    |
+| date | coptic_date         | Date picker with Coptic calendar        | coptic         |
+| date | ethiopian_date      | Date picker with Ethiopian calendar     | ethiopian      |
+| date | islamic_date        | Date picker with Islamic calendar       | islamic        |
+| date | bikhram_sambat_date | Date picker with Bikram Sambat calendar | bikhram_sambat |
+| date | myanmar_date        | Date picker with Myanmar calendar       | myanmar        |
+| date | persian_date        | Date picker with Persian calendar       | persian        |
```

Comment:
Can you rather split this into two sections, one focused on the three different "appearances" and the other just on the different calendar types, something like:

**Date picker appearances**

| type | name                | label                                   | appearance     |
| :--- | :------------------ | :-------------------------------------- | :------------- |
| date | rains_start         | When did the planting rains start?      | month-year     |
| date | year_migrate        | In which year did you migrate?          | year           |
| date | no-calendar_date    | Date picker without calendar            | no-calendar    |

**Supported non-Gregorian calendars**

| type | name                | label                                   | appearance     |
| :--- | :------------------ | :-------------------------------------- | :------------- |
| date | coptic_date         | Date picker with Coptic calendar        | coptic         |
| date | ethiopian_date      | Date picker with Ethiopian calendar     | ethiopian      |
| date | islamic_date        | Date picker with Islamic calendar       | islamic        |
| date | bikhram_sambat_date | Date picker with Bikram Sambat calendar | bikhram_sambat |
| date | myanmar_date        | Date picker with Myanmar calendar       | myanmar        |
| date | persian_date        | Date picker with Persian calendar       | persian        |

Reply:
I think that's a good approach. I will do that

Reply:
Done